akka-projection-jdbc/src/test/resources/logback-test.xml

```diff
@@ -14,6 +14,7 @@
         <appender-ref ref="STDOUT"/>
     </logger>
 
+    <logger name="akka.projection.jdbc" level="DEBUG"/>
 
     <root level="WARN">
```

Comment:
why not use DEBUG here? if something fails we want as much as possible

Reply:
sure